packages/react-data-grid-addons/src/shared/html5DragDropContext.js

```diff
@@ -0,0 +1,4 @@
+import { DragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+
+export default DragDropContext(HTML5Backend);
```

Comment:
`react-dnd` throws and an error if `DragDropContext` is initialized with `HTML5Backend` twice